### PR TITLE
Add openSenseMap-API

### PR DESCRIPTION
**Pull request by philippG777**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [ ] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by philippG777** [comment]
I added the openSenseMap-API at the end of the section, because it doesn't fit the category as well as the other APIs.

File: README.md [hunk 1]
```diff
@@ -959,5 +959,6 @@ API | Description | Auth | HTTPS | CORS |
 | [Storm Glass](https://stormglass.io/) | Global marine weather from multiple sources | `apiKey` | Yes | Yes |
 | [Weatherbit](https://www.weatherbit.io/api) | Weather | `apiKey` | Yes | Unknown |
 | [Yahoo! Weather](https://developer.yahoo.com/weather/) | Weather | `apiKey` | Yes | Unknown |
+| [openSenseMap](https://api.opensensemap.org/) | Data from Personal Weather Stations called senseBoxes | No | Yes | Yes |
```
Comment (yannbertrand) [inline README.md]: Hey, thank you for that contribution! 🎉 

It's OK to put in alphabetical order 😉 
```suggestion
| [openSenseMap](https://api.opensensemap.org/) | Data from Personal Weather Stations called senseBoxes | No | Yes | Yes |
| [OpenUV](https://www.openuv.io) | Real-time UV Index Forecast | `apiKey` | Yes | Unknown |
| [OpenWeatherMap](http://openweathermap.org/api) | Weather | `apiKey` | No | Unknown |
| [Storm Glass](https://stormglass.io/) | Global marine weather from multiple sources | `apiKey` | Yes | Yes |
| [Weatherbit](https://www.weatherbit.io/api) | Weather | `apiKey` | Yes | Unknown |
| [Yahoo! Weather](https://developer.yahoo.com/weather/) | Weather | `apiKey` | Yes | Unknown |
```

Comment (yannbertrand) [reply README.md]: Woops sorry my suggestion was not good...
I'm not sure I can fix it on GitHub directly, my suggestion duplicated those 5 links

**Comment by matheusfelipeog** [comment]
OpenUV, OpenWeatherMap, Storm Glass and Weatherbit APIs are already on README.md.

[Yahoo! Weather](https://developer.yahoo.com/weather/):
image

I will send openSenseMap to the master branch and close this PR.

**Pull request closed without merging**